src/Module/Settings/UserExport.php

```diff
@@ -42,6 +43,7 @@ public static function content()
 		$options = [
 			['settings/userexport/account', L10n::t('Export account'), L10n::t('Export your account info and contacts. Use this to make a backup of your account and/or to move it to another server.')],
 			['settings/userexport/backup', L10n::t('Export all'), L10n::t("Export your accout info, contacts and all your items as json. Could be a very big file, and could take a lot of time. Use this to make a full backup of your account \x28photos are not exported\x29")],
+			['settings/userexport/contactcsv', L10n::t('Export Contacts to CSV'), L10n::t("Export the list of the accounts you are following as CSV file. Compatible to e.g. Mastodon.")],
```

Comment:
Please rename to `contacts` if we don't have multiple export formats.

Reply:
That is something I need to investigate today, if e.g. Diaspora* has some import/export mechanism I would like to add that format as well.

Reply:
Fair enough.

Reply:
I've renamed it, as I've not found other platforms exporting the followed accounts.